### PR TITLE
Fix some uncaught errors, and other little fixes

### PR DESCRIPTION
**Pull request by VSEphpbb**

### CODE REVIEW

File: controller/admin_controller.php [hunk 1]
```diff
@@ -338,7 +338,7 @@ public function edit_rule($rule_id)
 		// Set output vars for display in the template
 		$this->template->assign_vars(array(
 			'S_EDIT_RULE'	=> true,
-			'S_IS_CATEGORY'	=> ($entity->get_right_id() - $entity->get_left_id() > 1) ? true : false,
+			'S_IS_CATEGORY'	=> $entity->get_right_id() - $entity->get_left_id() > 1,
```
Comment (rxu) [inline controller/admin_controller.php]: Does it mean ```($entity->get_right_id() - $entity->get_left_id()) > 1```? Just for readability.

**Comment by rxu** [comment]
Looks good.

File: controller/admin_controller.php [hunk 2]
```diff
@@ -124,8 +124,8 @@ public function display_options()
 
 		// Set output vars for display in the template
 		$this->template->assign_vars(array(
-			'S_ERROR'	=> (sizeof($errors)) ? true : false,
-			'ERROR_MSG'	=> (sizeof($errors)) ? implode('<br />', $errors) : '',
+			'S_ERROR'	=> sizeof($errors),
```
Comment (michaelcullum) [inline controller/admin_controller.php]: Could cast to a bool?

**Comment by VSEphpbb** [comment]
Updated

**Pull request merged**